### PR TITLE
feature: eth send helper contract

### LINKED ISSUE #402: helper contract for sending native asset (ETH) over the Bridge Xapp
should wrap the asset in WETH, approve the token bridge, and initiate the transfer

### PR DESCRIPTION
**Pull request by prestwich**
A simple contract to help users send ETH or other native assets over the bridge. Simply wraps the token in WETH and forwards it on

closes #402

### CODE REVIEW

**Comment by prestwich** [comment]
Deploy scripts for the helper contract blocked by #512

**Review by asaj — APPROVED** [review]
LGTM

**Comment by prestwich** [comment]
thanks, will hold off on merging until deploy scripts can be written

**Comment by prestwich** [comment]
@anna-carroll the deploy scripts have been written. this is ready for review and merge

File: solidity/optics-xapps/contracts/bridge/ETHHelper.sol [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.6.11;
+
+import {BridgeRouter} from "./BridgeRouter.sol";
+
+import {TypeCasts} from "@celo-org/optics-sol/contracts/XAppConnectionManager.sol";
+
+interface WETH {
```
Comment (anna-carroll) [inline solidity/optics-xapps/contracts/bridge/ETHHelper.sol:8]: can we move this to `interfaces`?

File: typescript/optics-deploy/src/bridge/index.ts [hunk 2]
```diff
@@ -97,6 +101,27 @@ async function deployBridgeRouter(deploy: BridgeDeploy) {
...
+
+  deploy.contracts.ethHelper = await factory.deploy(
+    deploy.config.weth!,
+    deploy.contracts.bridgeRouter?.proxy.address!,
+  );
+  await deploy.contracts.ethHelper.deployTransaction.wait(5);
+}
+
 /**
  * Enroll all other chains' BridgeRouters as remote routers
```
Comment (anna-carroll) [inline typescript/optics-deploy/src/bridge/index.ts:126]: we need to update writeDeployOutput to add the `ethHelper` and the `bridgeToken`

**Review by anna-carroll — APPROVED** [review]
LGTM :)

**Pull request merged**